### ISSUE TITLE
POSSIBLE BUG Lack of annotation/desc in 2nd and next URL requests

### ISSUE THREAD

**Issue by miglanc** [mendhak/gpslogger#479]
**TL;DR:**

When using option "Log to custom URL" the annotation/description parameter is being sent only once. All the following requests lacks of this parameter.

**Full:**

When I click a pencil icon again to edit/add the annotation again then the annotation that I have set a few seconds ago is missing too. Setting annotation again makes the next request to contain the parameter - but the same as before - only within a single request. In all following requests the parameter is missing.

Is this a bug or rather a feature? I mean, is this an expected behaviour? I rather find it as a bug. I am using version 80 on Samsung Galaxy Note 4 with Android 6.0.1.

*Despite this, @mendhak, congratulations! I like the app very much and is very useful for me, I think it is just awesome:) It is a great pleasure for me to use it!*

**Comment by mendhak** [comment]
Hi, thanks! This behavior is by design - just as an annotation would only be written for one point in a wirtten log file, the annotation is just sent once and the description gets cleared.

**Comment by miglanc** [comment]
Sorry, I was planning to write to you today to close this issue:)

> just as an annotation would only be written for one point in a wirtten log file, the annotation is just sent once and the description gets cleared.

I have thought about it a little it last days and I think that the current behavior is very good, it it much better than what I was trying to suggest, as it gives you much more opportunities how to handle the data by the system that is receiving the logging. Thanks!
